app/src/main/kotlin/com/saltedge/authenticator/features/main/MainActivityViewModel.kt

```diff
@@ -154,7 +163,7 @@ class MainActivityViewModel(
     }
 
     fun onUnlock() {
-        if (!initialQrScanWasStarted && connectionsRepository.isEmpty()) {
+        if (!initialQrScanWasStarted && interactorV1.noConnections && interactorV2.noConnections) {
```

Comment:
`interactorV1.noConnections` and `interactorV2.noConnections` both return the same result, always